### PR TITLE
Add documents to projects

### PR DESCRIPTION
**Pull request by jprokay**
I have completed the initial feature work. I am going to work on cleaning up debugging statements. 
I am unsure if it was the right call to duplicate the `FilesController` and to have moved the `files` directory into circles. 
#456

### CODE REVIEW

**Comment by dottorer** [comment]
@jprokay Can't compile PR-456 to a review app, the build seems to fail, I will try later, but does this work on your end? We might have some systemic issues with build dependencies though.

@phillipoertel : Does this look like a ruby issue on the build side?

**Comment by jprokay** [comment]
@dottorer I didn't have any issues on my side with starting the server. I will confirm that that is the case

**Comment by jprokay** [comment]
@dottorer @phil-monroe can we try another deploy of this?

**Comment by dottorer** [comment]
@jprokay Phil is no longer on Lale but I will look at a deploy on Monday. Thanks again for your great contribution!

**Comment by mkristian** [comment]
@jprokay could you please just squash the last three commits into one and revert the change on the Gemfile.lock (which is likely the cause of our problem compiling the slug on heroku). if you do not have time, I can do it the moment I come to this PR this week. thanks

**Comment by jprokay** [comment]
@mkristian will do it today. Thanks

**Comment by jprokay** [comment]
Did what I could to fix things up. If there is a better way to handle the commit squashing, let me know

File: config/application.rb [hunk 1]
```diff
@@ -56,6 +56,7 @@ class Application < Rails::Application
     config.cache_store = :memory_store, { size: 64.megabytes }
 
     if ENV['AWS_ACCESS_KEY'].present?
+      Rails.logger.warn "Setting up AWS"
```
Comment (mkristian) [inline config/application.rb]: @jprokay this line is causing the problem with the heroku deploy. please remove it.

**Comment by mkristian** [comment]
@jprokay thanks for cleaning up.

**Comment by mkristian** [comment]
hey @jprokay just 'stole' your PR and fixed this one line myself and now we hopefully get the review app deployed. I also made one single big commit out of your many commits - looks nicer on commit history and they all belong together from conceptual point of view. thank you very much for you contribution and please follow #463 for further progress info.

**Comment by mkristian** [comment]
closing in favor of #463

**Pull request closed without merging**